# Sets a new robot controller and destroys the old.
string sessionname
int32 sessionid

uint32 bodyid

# name used to query a controller
string controllername

# the arguments to ControllerBase::Init
string controllerargs
---
uint8 success
